buildroot-external/kernel/v6.6.y/hassos.config

```
@@ -43,6 +43,11 @@ CONFIG_SQUASHFS=y
 CONFIG_SQUASHFS_XATTR=y
 CONFIG_SQUASHFS_LZ4=y
 CONFIG_BTRFS_FS=m
+CONFIG_NTFS_FS=m
+CONFIG_NTFS3_FS=m
```

```suggestion
CONFIG_NTFS3_FS=m
```

`NTFS_FS` is the legacy driver and is fully replaced by `NTFS3_FS` in Linux 6.9+. It's rather pointless to have both options in HAOS, so I'll keep just the modern one.